collectors/lib/hadoop_http.py

```diff
@@ -32,20 +35,18 @@ class HadoopHttp(object):
     def __init__(self, service, daemon, host, port, uri="/jmx"):
         self.service = service
         self.daemon = daemon
-        self.port = port
-        self.host = host
-        self.uri = uri
-        self.server = httplib.HTTPConnection(self.host, self.port)
-        self.server.auto_open = True
+        self.url = "http://" + host + ":" + str(port) + uri
```

Comment:
NIT: I usually prefer join to + like ''.join([ 'http://' , host, ':', port, uroi])